includes/class-wc-order-item-meta.php

```diff
@@ -163,11 +163,13 @@ public function get_formatted( $hideprefix = '_' ) {
 	 * Return an array of formatted item meta in format e.g.
 	 * Handles @deprecated args.
 	 *
+	 * @deprecated 4.4.0
 	 * @param string $hideprefix Hide prefix.
 	 *
 	 * @return array
 	 */
 	public function get_formatted_legacy( $hideprefix = '_' ) {
+		wc_deprecated_function( 'WC_Order_Item_Product::get_formatted_legacy', '4.4.0', '' );
```

Comment:
We can't throw messages here, since we use it internally:

https://github.com/woocommerce/woocommerce/blob/ee158ae044db3822af6c39324df9175930ee0444/includes/class-wc-order-item-meta.php#L126-L129